app/controllers/Common.scala

```diff
@@ -3,7 +3,7 @@ package controllers
 import play.api.mvc._
 
 trait Common {
-  self: Controller =>
+  self: BaseController =>
```

Comment:
Actually I don't think we need the self type here. Are we using anything from Controller?